### PR TITLE
Added Task list to Readme.md

### PR DESCRIPTION
**Pull request by Raghav2305**
## Fixes Issue

This PR fixes the following issues:

closes #issue-number

## Changes proposed

Here comes all the changes proposed through this PR

## Check List (Check all the boxes which are applicable) 

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Screenshots 
Add all the screenshots which support your changes

### CODE REVIEW

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]
@Raghav2305 I think you misunderstood the issue. We want to build the readme for the project and the task list in the issue is there to help you know all things you need to add. It seems like you copy pasted the things that were present in the issue description. Please make the changes accordingly

File: README.md [hunk 1]
```diff
@@ -1 +1,8 @@
-Website
+- [ ] Tilte
```
Comment (shivam-sharma7) [inline README.md:1]:
```suggestion
- [ ] Tittle
```
typo

**Review by shivam-sharma7 — CHANGES_REQUESTED** [review]
I think here should be `Tittle` instead `Tilte`

**Comment by kaiwalyakoparkar** [comment]
No changes, updates or conversation initiated. Hence closing :)

**Pull request closed without merging**